ambry-router/src/main/java/com.github.ambry.router/PutOperation.java

```diff
@@ -1193,11 +1194,12 @@ void poll(RequestRegistrationCallback<PutOperation> requestRegistrationCallback)
      * for sending if this blob is composite, or marking the operation complete if this is a simple blob.
      */
     private void finalizeMetadataChunk() {
+      // @todo: set encrypted flag as appropriate.
```

Comment:
MetadataChunk will never be encrypted. So you can remove this todo

Reply:
Done.